### PR TITLE
Add copy button for ISBN

### LINKED ISSUE #2892: UX improvement: ISBN "Copy" button and dashed formatting
**Describe the proposition**
1. I propose a small COPY icon button close to ISBN number to copy it (see a red area on the screenshot):
2. Also, I would love to see formatted ISBNs for easier reading. This can be [done with JavaScript](http://inventaire.github.io/isbn3/?isbn=9781781682135).

**Expected behavior**

**Screenshots**
Screenshot 2023-07-05 at 12 48 22

**Instance**
Main instance.

**Additional context**

### PR DESCRIPTION
**Pull request by axiomizer**
For #2892

### CODE REVIEW

**Comment by mouse-reeve** [comment]
I think it might be easier to bring the hyphenation in a separate PR -- it's going to take me a minute to wrap my head around it, and it's going to need some revisions and it would be a shame to hold up the copy link because of that.

**Comment by axiomizer** [comment]
Thanks for taking a look :)

File: bookwyrm/templates/book/book_identifiers.html [hunk 1]
```diff
@@ -4,42 +4,48 @@
...
 <dl>
     {% if book.isbn_13 %}
-        <div class="is-flex">
+        <div class="is-flex is-flex-wrap-wrap">
             <dt class="mr-1">{% trans "ISBN:" %}</dt>
-            <dd itemprop="isbn">{{ book.isbn_13 }}</dd>
+            <dd itemprop="isbn" class="mr-1" id="isbn_content">{{ book.isbn_13 }}</dd>
+            <div>
+                <button class="button is-small" data-copywithtooltip data-content-id="isbn_content" data-tooltip-id="isbn_tooltip">
+                    <span class="icon icon-copy" title="{% trans "Copy ISBN" %}"></span>
```
Comment (mouse-reeve) [inline bookwyrm/templates/book/book_identifiers.html:12]: As is, this button is unlabelled for me when I use my screen reader. You can add a span that contains text for the screen reader to the icon's span using the class `is-sr-only` (ie, is screen reader only), something like:
```
<span class="icon icon-copy" title="{% trans "Copy ISBN" %}">
    <span class="is-sr-only">{% trans "Copy ISBN" %}</span>
</span>
```

File: bookwyrm/templates/book/book_identifiers.html [hunk 2]
```diff
@@ -4,42 +4,48 @@
...
+                </button>
+                <span class="copy-tooltip" id="isbn_tooltip">{% trans "Copied ISBN!" %}</span>
```
Comment (mouse-reeve) [inline bookwyrm/templates/book/book_identifiers.html:14]: This worked well for me on my screen reader

**Review by mouse-reeve — CHANGES_REQUESTED** [review]
Once this is screen reader friendly and the checks are passing (which we talked about in the chat) it should be good to go. Thank you so much for adding it!

File: bookwyrm/templates/book/book_identifiers.html (hunk 1 above)
Comment (axiomizer) [reply bookwyrm/templates/book/book_identifiers.html]: Sure it seems to work with the chromevox extension on my browser, but I added that span you wanted so hopefully that should work on your reader too now.

Comment (mouse-reeve) [reply bookwyrm/templates/book/book_identifiers.html]: Ah, how fickle the different brands of screen reader are! I'll have to give chromevox a try, I test with Apple's built-in VoiceOver

**Review by mouse-reeve — APPROVED** [review]
Thank you so much for adding this nice feature and being so patient with the janky CI and various tweaks in the review process!

**Comment by axiomizer** [comment]
My first merged PR 😊 Thanks for your help

**Comment by soshial** [comment]
Thank you so much, @axiomizer, for your efforts!
ISBNs looks is so much better now!

**Pull request merged**